web/template/certificate_provider_who_is_eligible.gohtml

```diff
@@ -0,0 +1,40 @@
+{{ template "page" . }}
+
+{{ define "pageTitle" }}{{ tr .App "whoIsEligibleToBeCertificateProvider" }}{{ end }}
+
+{{ define "main" }}
+    <div class="govuk-notification-banner govuk-notification-banner--success" role="alert"
+         aria-labelledby="govuk-notification-banner-title"
+         data-module="govuk-notification-banner">
+        <div class="govuk-notification-banner__header">
+            <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+                {{ tr .App "success" }}
+            </h2>
+        </div>
+        <div class="govuk-notification-banner__content">
+            <h3 class="govuk-notification-banner__heading">
+                {{ tr .App "weHaveIdentifiedYourCertificateProviderReferenceNumber" }}
+            </h3>
+        </div>
+    </div>
+
+    <h1 class="govuk-heading-xl">
+        {{ tr .App "whoIsEligibleToBeCertificateProvider" }}
+    </h1>
+
+    {{ $donorFullNamePossessive := .App.Localizer.Possessive .Lpa.Donor.FullName }}
```

Comment:
this is probably going to get some use, maybe worth adding a `possess` function next time

Reply:
Agree - I was going to add a ticket to go back and update everywhere we are doing this in templates.